docs/versioned_docs/version-2.x/components.mdx

```diff
@@ -724,14 +748,53 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
 
 * **Configuration**
 
-  Include [HFTransformersNLP](./components.mdx#hftransformersnlp) and [LanguageModelTokenizer](./components.mdx#languagemodeltokenizer) components before this component. Use
-  [LanguageModelTokenizer](./components.mdx#languagemodeltokenizer) to ensure tokens are correctly set for all components throughout the pipeline.
+  :::caution Changed in 2.1
+  TODO: do we need to explain the config changes here?
```

Comment:
I think it's fine without the changes listed here

Reply:
how about moving the box at the end of the section and say:
```md
:::caution Changed in 2.1
The `model_name`, `model_weights` and `cache_dir` parameters were added to the pipeline configuration file.
:::
```
?

Reply:
ah I didn't realize that was new. Yes let's add that